deployer/README.md

```diff
@@ -17,172 +17,191 @@ pip install --editable .
+
+The `deployer.py` file is the main file, that contains all of the commands registered directly on the `deployer` main typer app, that could not or were not yet categorized in sub-commands.
 
 ```bash
- Usage: deployer deploy [OPTIONS] CLUSTER_NAME [HUB_NAME]                                                               
-                                                                                                                        
- Deploy one or more hubs in a given cluster                                                                             
-                                                                                                                        
-╭─ Arguments ──────────────────────────────────────────────────────────────────────────────────────────────────────────╮
-│ *    cluster_name      TEXT        Name of cluster to operate on [default: None] [required]                          │
-│      hub_name          [HUB_NAME]  Name of hub to operate deploy. Omit to deploy all hubs on the cluster             │
-│                                    [default: None]                                                                   │
-╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
-╭─ Options ────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
-│ --dask-gateway-version        TEXT  Version of dask-gateway to install CRDs for [default: 2023.1.0]                  │
-│ --debug                             When present, the `--debug` flag will be passed to the `helm upgrade` command.   │
-│ --dry-run                           When present, the `--dry-run` flag will be passed to the `helm upgrade` command. │
-│ --help                              Show this message and exit.                                                      │
-╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+├── commands
+│   ├── cilogon_client_cmd.py
+│   ├── deployer.py
+│   ├── exec
+│   │   ├── debug_app_and_commands.py
+│   │   └── shell
+│   │       ├── app.py
+│   │       ├── cloud_commands.py
+│   │       └── infra_components_commands.py
+│   ├── generate
+│   │   ├── __init__.py
+│   │   ├── billing
+│   │   │   ├── cost_table_cmd.py
+│   │   │   ├── importers.py
+│   │   │   └── outputers.py
+│   │   ├── dedicated_cluster
+│   │   │   ├── aws_commands.py
+│   │   │   ├── common.py
+│   │   │   ├── dedicate_cluster_app.py
+│   │   │   └── gcp_commands.py
+│   │   └── helm_upgrade
+│   │       ├── decision.py
+│   │       └── jobs_cmd.py
+│   ├── grafana
+│   │   ├── central_grafana.py
+│   │   ├── deploy_dashboards_cmd.py
+│   │   ├── tokens_cmd.py
+│   │   └── utils.py
+│   └── validate
+│       ├── cluster.schema.yaml
+│       └── config_cmd.py
 ```
 
-### `validate`
+### The `tests` directory
```

Comment:
While we're moving this around, let's call these 'health-check-tests' or something like that for clarity, that it's not testing the deployer itself?